### ISSUE TITLE
cross entropy loss

### ISSUE THREAD

**Issue by Chenly1993** [vlfeat/matconvnet#982]
I want to use the cross entropy loss function for the output of sigmoid function.How can I do?
Thank you

**Comment by XuYunqiu** [comment]
Also want to know how to deal with

**Comment by bazilas** [comment]
Yes. Just check the given examples (e.g. MNIST or CIFAR). You might need to add a sigmoid layer at the end of the network.
